README.md

```diff
@@ -5,6 +5,25 @@ By storing them in a central place (here), and distributing them in an automated
 1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
 2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
 
+## Customization
+
+Most repositories won't need any customization, and the workflows defined here will just work fine.
+Some repositories may require some pre-setup steps to be run before tests (or code checks) can be run. Setup steps for `go-test` are defined in `.github/actions/go-test_setup/action.yml`, and setup steps for `go-check` are defined in `.github/actions/go-check_setup/action.yml`, in the following format:
+
+```yml
+runs:
+  using: "composite"
+  steps:
+    - name: Step 1 
+      shell: bash
+      run: echo "do some initial setup"
+    - name: Step 2
+      shell: bash
+      run: echo "do some more advanced setup"
```

Comment:
I think this should also demo a package install on Linux with `if: ...`, as I think that will be fairly common

Reply:
I didn't expect it to work, but apparently we actually have access to the `matrix` variable in the local action, see https://github.com/protocol/.github-test-target/runs/3321274123.
I updated the README.

Reply:
:tada: